packages/@glimmer/debug/test/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Some of the test packages were missing `package.json`s for some reason, which broke the assumptions
in the linting setup.